### PR TITLE
Special requirements question

### PR DESCRIPTION
**Pull request by leoapost**
Introduce a new question for the applicant to describe any special requirements or particular need they may have for the hearing.

### CODE REVIEW

File: app/views/application/_multi_answer_row.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<div>
+  <dt class="question">
+    <%= translate_with_appeal_or_application "check_answers.#{multi_answer_row.question}.question" %>
+  </dt>
+
+  <dd class="answer">
+    <ul class="list list-bullet">
+      <% multi_answer_row.value.each do |value| %>
+        <% if value.is_a?(String) && value.present? %>
```
Comment (zaparka) [inline app/views/application/_multi_answer_row.html.erb]: shouldn't we move this to a helper?

File: app/views/application/_multi_answer_row.pdf.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<tr>
+  <td class="question">
+    <%= translate_with_appeal_or_application "check_answers.#{multi_answer_row.question}.question" %>
+  </td>
+
+  <td class="answer answer-multi-choice">
+    <ul class="list list-bullet">
+      <% multi_answer_row.value.each do |value| %>
+        <% if value.is_a?(String) && value.present? %>
```
Comment (zaparka) [inline app/views/application/_multi_answer_row.pdf.erb]: same as above

File: app/views/application/_multi_answer_row.html.erb (hunk 1 above)
Comment (leoapost) [reply app/views/application/_multi_answer_row.html.erb]: I've moved the type checking to a helper method

File: app/views/application/_multi_answer_row.pdf.erb (hunk 2 above)
Comment (leoapost) [reply app/views/application/_multi_answer_row.pdf.erb]: I've moved the type checking to a helper method

Comment (zaparka) [reply app/views/application/_multi_answer_row.pdf.erb]: cool thx.

**Comment by leoapost** [comment]
Danger was complaining about some commits messages and I had to rebase and force push the new commits. @zaparka @AlexaVB  can you please review again?

**Pull request merged**